src/hooks/useSubaccount.tsx

```diff
@@ -945,6 +946,7 @@ const useSubaccountContext = ({ localDydxWallet }: { localDydxWallet?: LocalWall
       mutationFn: async (affiliate: string) => {
         const tx = await registerAffiliate(affiliate);
         dispatch(removeLatestReferrer());
+        track(AnalyticsEvents.AffiliateRegistration({ affiliateAddress: affiliate }));
```

Comment:
[nit] can we rename `affiliate` to `affiliateAddress`? sometimes i get confused not knowing if things are strings or objects based on the names 😭